src/reconcile/rec_col.c

```diff
@@ -869,6 +869,13 @@ __wt_rec_col_fix(
             upd = NULL;
             /* Make sure not to apply an uninitialized time window, or one from another key. */
             WT_TIME_WINDOW_INIT(&unpack.tw);
+
+            /* Initialize the selected update. */
+            upd_select.upd = NULL;
```

Comment:
Should we have a macro to init those too? Given we already have one for the time window, we could have a `clear` macro that inits everything and calls `WT_TIME_WINDOW_INIT`.

Reply:
done